app/src/main/java/org/mozilla/fenix/collections/CollectionCreationView.kt

```diff
@@ -243,6 +239,10 @@ class CollectionCreationView(
         }
 
         view.name_collection_edittext.showKeyboard()
+        nameCollectionConstraints.clone(
```

Comment:
Not your fault and nothing you need to do but there are a few things I don't like about this file:
- It nests layouts dynamically using `.clone` and `ConstraintSet.applyTo`: the conventional way to do this is using `<include>` tags.
- Each layout it includes has an additional root `ConstraintLayout`, afaict by necessity of ^ method; this slows down inflation
- There is duplication across methods in the code (e.g. `updateForNameCollection` and `updateForRenameCollection`)